studio-ui/static-assets/themes/cstudioTheme/css/console.css

```
@@ -0,0 +1,1082 @@
+}
+
+.content-path-input--input {
+	width: 100%;
+	border: none;
+	display: block;
+	padding: 4px 18px 4px 2%;
+	border-bottom: 1px solid #f3f3f3;
+}
+
+.content-path-input--icon {
+	background-color: Transparent;
```

Stylelint (17.14.0)

[error] Expected "Transparent" to be "transparent" (value-keyword-case)

(value-keyword-case)